clean up error codes explanation (#68482)
rust-lang/rust: src/librustc_error_codes/error_codes/E0220.md

> A used associated type wasn't defined in the trait.

Reviewer: ```suggestion
An associated type was used which was not defined in the trait.
```

Author: This sentence makes no sentence. :p

> A used associated type wasn't defined in the trait.

Reviewer: ```suggestion
The associated type used was not defined in the trait.
```
